privacyidea/api/token.py

```diff
@@ -902,6 +904,7 @@ def loadtokens_api(filename=None):
         "oathcsv" or "yubikeycsv".
     :jsonparam tokenrealms: comma separated list of realms.
     :jsonparam psk: Pre Shared Key, when importing PSKC
+    :jsonparam pskcValidateMAC: Determines how invalid MACs should be handled when importing PSKC
```

Comment:
The API documentation is generated from the API docstrings. So it would be helpfull to add s.th. like
"Allowed values are 'no_check', 'check_fail_soft' and 'check_fail_hard'". Since this then will be in a prominent location at privacyidea.readthedocs.io.